Tasks/AzurePowerShellV4/InitializeAz.ps1

```diff
@@ -43,10 +43,10 @@ Write-Host "##[command]Import-Module -Name $($module.Path) -Global"
 $module = Import-Module -Name $module.Path -Global -PassThru -Force
 
 # Clear context
-Write-Host "##[command]Clear-AzContext -Scope Process"
-$null = Clear-AzContext -Scope Process
 Write-Host "##[command]Clear-AzContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue"
 $null = Clear-AzContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue 
+Write-Host "##[command]Clear-AzContext -Scope Process"
+$null = Clear-AzContext -Scope Process
```

Comment:
Have made this change because I found that context in scope of currentuser must be cleared before process to clear it fully , when I made change of if check in disconnect-azaccount